src/mixins/nft.js

```diff
@@ -85,6 +85,19 @@ export default {
         this.iscnId
       )}%2F1`;
     },
+    populatedEvents() {
+      return this.NFTHistory.map(event => ({
+        ...event,
+        toDisplayName: this.displayNameList[event.toWallet] || event.toWallet,
+      }));
+    },
+    populatedCollectors() {
+      return this.sortedOwnerListId.map(id => ({
+        id,
+        toDisplayName: this.displayNameList[id] || id,
```

Comment:
This can keep as `displayName`, since there is no from/to context
```suggestion
        displayName: this.displayNameList[id] || id,
```

Reply:
oh, I see what you mean now 🤣